kyuubi-server/src/main/scala/org/apache/kyuubi/sql/plan/trino/TrinoFeOperations.scala

```diff
@@ -15,14 +15,14 @@
  * limitations under the License.
  */
 
-package org.apache.kyuubi.sql.trino
+package org.apache.kyuubi.sql.plan.trino
 
 import org.apache.kyuubi.sql.plan.KyuubiTreeNode
 
 /////////////////////////////////////////////////////////////////////////////////////////
 // This file contains all Trino JDBC operation nodes which are parsed from statement
 /////////////////////////////////////////////////////////////////////////////////////////
 
-case class TrinoGetSchemas(catalogName: String, schemaPattern: String) extends KyuubiTreeNode {
+case class GetSchemas(catalogName: String, schemaPattern: String) extends KyuubiTreeNode {
```

Comment:
does this change necessary ? Seems Trinoxxx is more cleaer to me

Reply:
Kyuubi supports trino **engine** and trino **fe**, Trinoxxx seems to be ambiguous, or rename to TrinoFexxx?